### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by billsacks**
- [x] list item

- [ ] another item to do

- [ ] and one more

> - [ ] Does this turn into a check list?

>- >> + >1. - [ ] todo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ Here is a change
 # Another heading
 
 Another change
+
```
Comment (billsacks) [inline README.md:8]: Here's what I think.

File: README.md [hunk 2]
```diff
@@ -5,3 +5,5 @@ Here is a change
...
+One more change!
```
Comment (billsacks-test) [inline README.md:9]: Well here's what *I* think.

**Comment by billsacks** [comment]
- - 1. + * 2. [ ] todo

>- >> + >1. - [ ] todo

**Comment by billsacks** [comment]
- -[ ] this is a todo

- --[ ] this is not a todo

**Comment by billsacks** [comment]
[ ] what about this

  [ ] and this

**Comment by billsacks** [comment]
-   [ ] with extra spaces

**Comment by billsacks** [comment]
- [  ] two spaces in the brackets

**Comment by billsacks** [comment]
> - > - >> 1. [ ] todo

>>>>- [ ] todo

>- >- >- [ ] todo

>> - >> - [ ] todo

> > - > > - > > - [ ] todo

> - - > 1. 2. > - [ ] todo

**Comment by billsacks** [comment]
- [ ]todo

**Comment by billsacks** [comment]
- [ ] item 0

Hello
- [ ] item 1
- [ ] item 2

Goodbye
- [ ] item 3

That's all

**Pull request closed without merging**